config/buses/gcppubsub/README.md

```diff
@@ -3,10 +3,20 @@
 Deployment steps:
 1. Setup [Knative Eventing](../../../DEVELOPMENT.md)
 1. [Create a service account](https://console.cloud.google.com/iam-admin/serviceaccounts/project) with the 'Pub/Sub Editor' role, and download a new JSON private key.
-1. Create a secret for the downloaded key `kubectl create secret generic gcppubsub-bus-key --from-file=key.json=PATH-TO-KEY-FILE.json`
-1. Configure the bus, replacing `$PROJECT_ID` with your GCP Project ID, `kubectl create configmap gcppubsub-bus-config --from-literal=GOOGLE_CLOUD_PROJECT=$PROJECT_ID`
+1. Configure the bus. Change `$NAMESPACE` to the namespace for the bus, or `knative-eventing` for cluster buses:
+     1. Create a secret for the downloaded key:
+         ```
+         kubectl create secret generic gcppubsub-bus-key --namespace $NAMESPACE --from-file=key.json=PATH-TO-KEY-FILE.json
+         ```
+    1. Create a configmap, replacing `$PROJECT_ID` with your GCP Project ID: 
+        ```
+        kubectl create configmap gcppubsub-bus-config --namespace $NAMESPACE --from-literal=GOOGLE_CLOUD_PROJECT=$PROJECT_ID
+        ```
 1. For cluster wide deployment, change the kind in `config/buses/gcppubsub/gcppubsub-bus.yaml` from `Bus` to `ClusterBus`.
-1. Apply the 'gcppubsub' Bus `ko apply -f config/buses/gcppubsub/`
+1. Apply the 'gcppubsub' bus:
```

Comment:
It feels like this should document using the released bus YAML (https://github.com/knative/docs/tree/master/eventing#buses), with documentation about how to configure it.

(We'll also need some way to figure out how to push this documentation into that location when we actually start cutting releases.)